### ISSUE TITLE
Accuracy for the float baseline

### ISSUE THREAD

**Issue by Tiiiger** [boluoweifenda/WAGE#10]
Hi, 

I am very interested in your work. In the paper, you present a table comparing the performance of the low-precision model and the float model (Figure 3). I am not sure how to replicate the float result using the existing code base. So I want to ask what is the final test accuracy of the vanilla float model? 

Thank you in advance.

**Comment by boluoweifenda** [comment]
@Tiiiger 
For 28ff, just set bitG and bitE to 32, and change the optimizer to Momentum. Note that weights are scaled by the factor \alpha in the forward propagation, their updates are also decreased in the backpropagation. So please increase learning rate to overcome the overfitting problem.

For vanilla, set bitW, bitA, bitG, bitE to 32, and change the optimizer to Momentum. At this time, you can add weight decay (1e-4) and batch normalization to obtain better performance. The learning rate and annealing method is same as that in conventional CNNs, e.g., 0.1 for 80 epochs, 0.01 for 40 epochs and 0.001 for 40 epochs.

Good luck :  )
